components/misc_metrics/page_metrics.cc

```diff
@@ -242,4 +260,13 @@ void PageMetrics::OnDomainDiversityResult(
   VLOG(2) << "PageMetrics: domains loaded report, count = " << count;
 }
 
+void PageMetrics::ReportBookmarkCount() {
+  if (!bookmark_model_ || !bookmark_model_->loaded()) {
+    return;
+  }
+  int count = CountBookmarks(bookmark_model_->root_node());
```

Comment:
if we really don't like reusing the existing histogram, why don't we use`BookmarkCounter` here

Reply:
thanks, changed